packages/itwinui-css/backstop/tests/tag.html

```diff
@@ -46,49 +47,59 @@ <h2>Default</h2>
         >
           Tag
         </span>
-        <button
+        <span
```

Comment:
Changed `<button>` to `<span role="button">` which does not receive `:focus`.  `tabindex="0"` is set on the parent.